### PR TITLE
Added README example using "rake test" instead of rspec

### PR DESCRIPTION
**Pull request by jasnow**
Added README example using "rake test" instead of rspec

### CODE REVIEW

**Comment by jasnow** [comment]
Generalize the usage to show "rspec" and "test" examples.

File: README.md [hunk 1]
```diff
@@ -52,6 +53,13 @@ gelauto run --annotate $(find . -name '*.rb') -- bundle exec rspec spec/
 
 You can also choose to run Gelauto with the `--rbi` flag, which will cause Gelauto to print results to standard output in [RBI format](https://sorbet.org/docs/rbi).
 
+In this second example, we're going to be running an **/test** test suite.
```
Comment (camertron) [inline README.md:56]: Unless I'm much mistaken, the `test/` directory is the default for [minitest](https://github.com/seattlerb/minitest).

**Review by camertron — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (jasnow) [reply README.md:56]: "rails new {APP}" creates "test/" dir too. "find test" produces:
```
test
test/application_system_test_case.rb
test/controllers
test/controllers/.keep
test/fixtures
test/fixtures/.keep
test/fixtures/files
test/fixtures/files/.keep
test/helpers
test/helpers/.keep
test/integration
test/integration/.keep
test/mailers
test/mailers/.keep
test/models
test/models/.keep
test/system
test/system/.keep
test/test_helper.rb
```

Comment (camertron) [reply README.md:56]: Yep, true. That's because Rails uses minitest :) https://guides.rubyonrails.org/testing.html#rails-meets-minitest

Comment (jasnow) [reply README.md:56]: OK - You are welcome to reword my text.

**Comment by camertron** [comment]
@jasnow since I can't push to your copy of the repo, I'm closing this PR and opening a new one off origin.

**Pull request closed without merging**